gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,324 @@
+= Renewing Flex Gateway Registration
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Renewing Flex Gateway registration ensures the continued operation of your applications. You renew registration by using the `flexctl` CLI.
+
+To avoid potential disruptions, verify the success of the renewal process by using the `flexctl inspect` command.
+
+This document describes two `flexctl` CLI commands: `renew`, and `inspect`.
+
+* <<renewal-cli-overview,Registration Renewal CLI Command Overview>>
+* <<inspection-cli-overview,Registration Inspection CLI Command Overview>>
+
+== Before You Begin
+
+* xref:flex-install.adoc[Download Flex Gateway].
++
+To invoke the registration renewal `flexctl` command, download the latest version of Flex Gateway. It is not necessary to register and run this version of the gateway.
+
+[[renewal-cli-overview]]
+== Registration Renewal CLI Command Overview
+
+Flex Gateway includes the following CLI command to renew registration:
+
+----
+flexctl registration renew
+----
+
+The command renews an existing Flex Gateway registration with Runtime Manager. For authentication, you specify *either* user credentials or connected appl credentials. Authentication with user credentials requires the `--username` and `--password` flags. Authentication with connected app credentials requires the `--client-id` and `--client-secret` flags.
+
+Invoke the following `help` command for information about usage:
+
+[source,ssh]
+----
+flexctl registration renew --help
+----
+
+For Flex Gateway running in a container, invoke `help` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl mulesoft/flex-gateway renew --help
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl docker.io/mulesoft/flex-gateway renew --help
+----
+====
+
+Refer to the following topics for more information about renewal command usage:
+
+* <<renew-with-user-password,Renew Registration with User Credentials>>
+* <<renew-with-conn-app,Renew Registration with Connected App Credentials>>
+
+[[renew-with-user-password]]
+== Renew Registration with User Credentials
+
+Flex Gateway includes the following CLI command to renew registration using Anypoint Platform user credentials:
+
+[source,ssh]
+----
+flexctl registration renew --username=<your-username> --password=<your-password> \
+ <path-to-registration-file>
+----
+
+[NOTE]
+====
+The user must include _Read Servers_ and _Manage Servers_ permissions for Runtime Manager.
+====
+
+Replace `<path-to-registration-file>` with the path and filename of the existing `registration.yaml` file. Typically, the existing registration file is overwritten by the renewal process. You can optionally output the new registration file to a directory by using the `--output-directory` flag.
```

Comment:
the existing registration file is overwritten only if the output directory is the same one as the directory where the registration file is